README.md

```diff
@@ -206,3 +206,32 @@ After processing the envelope, the delivery service forwards the message to the
 **ERC3668Resolver**: 0x88c8cC822095cdE6F92c8d20311C8e7dE6A98694
 
 **SignatureVerifier**: 0xA76d65E1241CF77944AB66835Bb5389a3E53A269
+
+# Contributing
+
+We open feature branches from `develop` and merge them back via pull requests. The `main` branch holds the current production release.
+
+There are 3 remote environments where the code is deployed:
+
+1. `production` - the production environment that is automatically deployed from the `main` branch
+2. `staging` - the staging environment that is automatically deployed from the `develop` branch and is used to test and review the code before it is deployed to production
+3. `testing` - this environment is used to deploy feature branches people are working on and need to test in a real environment
+
+## Using the testing environment
+
+In order to deploy a branch to testing, do this:
+
+1. talk to the team to make sure nobody is actively using the testing environment at the moment
```

Comment:
Is it possible for people outside the dm3-org to trigger any workflow?

Reply:
Contributors like @Bhupesh-mfsi can use actions, but randos can't.